mapping.csv

```diff
@@ -400,6 +426,10 @@
 "^emudbot$",emudbot,
 "^vidro$",vidro,
 "^eitest$",eitest,
+"^dark(c|k)omet$",darkcomet,https://threats.kaspersky.com/en/threat/Backdoor.Win32.DarkKomet/
+"^redos(dru)?$",redosdru,
+"^(sourface|coresh(ell)?|sofacy|foosace)$",sofacy,
```

Comment:
Sofacy is a threat actor (https://malpedia.caad.fkie.fraunhofer.de/actor/sofacy 
machine-readable source: https://github.com/MISP/misp-galaxy/blob/main/clusters/threat-actor.json)

Reply:
it's also used to name related malware 
probably good to use common TA name 

https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Trojan:Win32/Sofacy.A!dha&threatId=-2147241509

Reply:
Mapping malware to actors is different to the current purpose (mapping malware to malware families). so that'd be a scope change. Doesn't such a mapping already exist elsewhere? I see that malpedia has some mappings (but doesn't map the names you mention, presumably because they don't have the artifact, which malpedia requires for each malware), MITRE has some info as well, but I don't know if that's machine readable.

Reply:
changed